mcs/class/monodoc/Monodoc.Ecma/EcmaDesc.cs

```diff
@@ -280,6 +281,25 @@ void ConstructCRef (StringBuilder sb)
 			}
 		}
 
+		void AppendGenericArguments (StringBuilder sb, IEnumerable<EcmaDesc> arguments, bool isNumeric, int argumentsCount)
+		{
+			if (arguments != null && isNumeric) {
+				sb.AppendFormat ("`{0}", argumentsCount);
+			} else if (arguments != null) {
+				sb.Append ('<');
```

Comment:
Given that the provision of type arguments within CREFs is a non-standard extension anyway, perhaps we should just use/standardize on `{}` instead of `<>`? These values are often stored in XML, and using `<>` will require escaping them, while `{}` won't.